elementary/clients/dbt/subprocess_dbt_runner.py

```
@@ -26,16 +26,17 @@ def _inner_run_command(
     ) -> DbtCommandResult:
         try:
             result = subprocess.run(
```

Ruff (0.13.1)

`subprocess` call: check for execution of untrusted input

(S603)